app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.styles.tsx

```diff
@@ -0,0 +1,44 @@
+import { css } from '@emotion/css';
+
+export const connectionPanelContainer = css`
+	border-radius: 8px;
+	box-shadow: 0px 2px 4px 0px #091E4240;
+	display: flex;
+	flex-direction: column;
+	max-height: 164px;
+	padding: 1.5em;
```

Comment:
There's an Atlassian-wide movement towards [Space tokens](https://atlassian.design/foundations/spacing#space-tokens). It'd be good if we implemented them for any new features we are creating :)